### ISSUE TITLE
axe (accessibility audit) - dynamic content not announced

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/olo.sk#861]
- Dynamic content not announced - modal, pop-ups are not announced to screen readers
- Use ARIA live regions (aria-live=“polite”)

<img width="1285" alt="Screenshot 2024-11-05 at 16 24 02" src="https://github.com/user-attachments/assets/eb5d1eae-3dab-4c8e-a6b4-d51a70d63142">
